python/pyarrow/_hdfs.pyx

```diff
@@ -93,9 +93,10 @@ cdef class HadoopFileSystem(FileSystem):
         Instantiate HadoopFileSystem object from an URI string.
 
         The following two calls are equivalent
-        * HadoopFileSystem.from_uri('hdfs://localhost:8020/?user=test'
-                                    '&replication=1')
-        * HadoopFileSystem('localhost', port=8020, user='test', replication=1)
+
+        * ``HadoopFileSystem.from_uri('hdfs://localhost:8020/?user=test&replication=1')`` # noqa: E501
+
+        * ``HadoopFileSystem('localhost', port=8020, user='test', replication=1)`` # noqa: E501
```

Comment:
Anyone have any opinion on ignoring flake here?  RST didn't like the multi line strings.  I tried using literal blocks `::` but then flake complained about the empty lines required by RST.  This seems like the most readable but the lines are too long (hence the `# noqa`)

Reply:
I'm fine with this.

Reply:
I added a suggestion below to fix this. I think that's preferrable since the ``# noqa: E501`` actually would show up as normal text which would be strange in the online rendered docstring